code/tim/HTML+CSS/lab01_Bio/index.html

```diff
@@ -0,0 +1,63 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Brendan Fraser</title>
+    <link rel="stylesheet" href="./styles.css">
+</head>
+<body>
+    
+    <div id="bio_info">
+    <h2>Brendan Fraser</h2>
+    <h3>Brendan James Fraser was born in Indianapolis, Indiana, to Canadian parents Carol Mary (Genereux), a sales counselor, and Peter Fraser, a journalist and travel executive. He is of Irish, Scottish, German, Czech, and French-Canadian ancestry. As his parents frequently moved, Brendan can claim affinity with Ottawa, Indianapolis, Detroit, Seattle, London and Rome. His early exposure to theatre, particularly in London, led him to Seattle's Cornish Institute.</h3>
```

Comment:
Consider using a p tag rather than a header h3 tag. This section is not centered.